org.opent2t.sample.lamp.superpopular/com.contosothings.lightbulb/js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "opent2t-translator-com-contosothings-lightbulb",
+    "version": "0.0.2",
+    "description": "ContosoThings Light Bulb",
+    "homepage": "http://opentranslatorstothings.org",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/opent2t/translators.git"
+    },
+    "main": "thingTranslator.js",
+    "license": "MIT",
+    "scripts": {
+        "test": "ava"
+    },
+    "ava": {
+        "files": [
+            "tests/*.js"
+        ]
+    },
+    "dependencies": {
+        "opent2t-translator-com-contosothings-hub": "^0.0.1"
+    },
+    "devDependencies": {
+        "opent2t": "^1.0.4",
```

Comment:
This needs to be dependency now that we use Opent2TError + OpenT2Tlogger classes